docs/src/pages/components/grid/grid.md

```diff
@@ -14,6 +14,8 @@ Material Design’s responsive UI is based on a 12-column grid layout.
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
+> ⚠️ The `Grid` component shouldn't be confused with a data grid. The component is closer to a layout grid. For a data grid head to [our `DataGrid` component](/components/data-grid/).
```

Comment:
data-grid?

Reply:
I'm not aware of any prior usages of the hyphenated version in our docs. What's the advantage?

Reply:
```suggestion
> ⚠️ The `Grid` component shouldn't be confused with a data grid; it is closer to a layout grid. For a data grid head to [the `DataGrid` component](/components/data-grid/).
```